__tests__/app.test.js

```diff
@@ -76,6 +76,64 @@ describe('GET /api/articles', () => {
   });
 });
 
+describe('GET /api/articles?sort_by', () => {
+  test('200: Responds with an array sorted by article_id, default descending order', () => {
+    return request(app)
+      .get('/api/articles?sort_by=article_id')
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles.length).toBe(5);
+        expect(articles).toBeSortedBy('article_id', { descending: true });
+      });
+  });
+  test('200: Responds with an array sorted by title, default descending order', () => {
+    return request(app)
+      .get('/api/articles?sort_by=title')
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles.length).toBe(5);
+        expect(articles).toBeSortedBy('title', { descending: true });
+      });
+  });
+  test('400: Responds with an error message if invalid sort_by query', () => {
+    return request(app)
+      .get('/api/articles?sort_by=mouthfeel')
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe('Bad Request');
+      });
+  });
+});
+
+describe('GET /api/articles?order', () => {
+  test('200: Responds with an array sorted in ascending order, default sort_by query (created_at)', () => {
+    return request(app)
+      .get('/api/articles?order=ASC')
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles.length).toBe(5);
+        expect(articles).toBeSortedBy('created_at', { ascending: true });
+      });
+  });
+  test('200: Responds with an array sorted in ascending order, custom (valid) sort_by query', () => {
+    return request(app)
+      .get('/api/articles?sort_by=article_id&order=ASC')
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles.length).toBe(5);
+        expect(articles).toBeSortedBy('article_id', { ascending: true });
+      });
+  });
+  test('400: Responds with an error message if invalid order query', () => {
+    return request(app)
+      .get('/api/articles?order=shuffle')
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe('Bad Request');
+      });
+  });
+});
+
```

Comment:
Ah, you're right! I've done a quick pass and updated 19 tests to use destructured `{ body: { msg } }` rather than dot notation, consistency is key! :saluting_face: